ycb_models:
  names:
    - mug
    - wood_block
    - master_chef_can
